rfcs/text/0001-rfc-process.md

```diff
@@ -181,7 +203,7 @@ Where necessary, existing RFCs can be revised using the same process (a proposed
 
 - Validate that the RFC pull request was successfully updated and merged
```

Comment:
Authors seem to struggle with remembering to update the PR link in the template and remove the backticks. Might mention here.